hadoop-ozone/common/src/main/bin/ozone

```diff
@@ -124,7 +124,7 @@ function ozonecmd_case
     ;;
     freon)
       HADOOP_CLASSNAME=org.apache.hadoop.ozone.freon.Freon
-      OZONE_FREON_OPTS="${OZONE_FREON_OPTS} -Dhadoop.log.file=ozone-freon.log -Dlog4j.configuration=file:${ozone_shell_log4j}"
+      OZONE_FREON_OPTS="${OZONE_FREON_OPTS}"
```

Comment:
Not sure if this can achieve the description in the JIRA. Do we miss some additional options?

Reply:
Without the specific ozone_shell_log4j configuration the default log4j would be used which prints out everything to the stdout on INFO level.

Reply:
Make sense to me. +1.